### DISCUSSION TITLE
direction of the arrow between commits, is it not confusing?

### DISCUSSION THREAD

**Discussion by filunieto** [pcottle/learnGitBranching#992] [General]
the turorial is simply fantastic, but there is one thing that I don't see clearly or maybe I don't understand.
When you draw the commit C0 and C1, for example, the direction of the arrow goes from C1 to C0,
shouldn't it be the opposite direction? i attache an image
Thanks
<img width="1548" alt="Screen Shot 2022-08-05 at 11 14 21 AM" src="https://user-images.githubusercontent.com/42672484/183045169-da3c1add-ddbd-43f9-bb04-af711a496b79.png">

**Comment by mgarciaisaia** [comment]
That's how git really works. `C0` doesn't know a thing about `C1` - it's `C1` that has a pointer to its parent commit.

So I guess making that point to users/learners is part of what the site wants to (and probably should) do?

**Comment by pcottle** [comment]
Exactly as @mgarciaisaia mentioned! I think it's somewhat important to understand that if you're at `C0`, you can't ask git "oh just please move forward to C1", because there's no concept of "forward"! `C0` could have multiple commits pointing into it, as well as it has no reference to C1, so you really need to manage your position with branches (and a bit trickier, reflogs).

You are the one who decides that "forward" means C1 from C0, in a sense :) 

But yeah although this is a bit surprising at first, I think it's important to match the model of how git actually works under the hood
